cmd/state-offline-installer/main.go

```diff
@@ -63,52 +61,23 @@ func run() error {
 
 	params := newParams()
 	cmd := captain.NewCommand(
-		path.Base(os.Args[0]), "", "", p, nil, nil,
+		cmdInstall,
```

Comment:
Note; for clarity this should probably match the name of the executable. eg. "offline-installer".